src/components/shared/ArrowPanel.js

```diff
@@ -15,27 +15,27 @@ type Props = {
   onOkButtonClick?: () => mixed,
   onCancelButtonClick?: () => mixed,
   className: string,
-  children: React$Element<*>,
+  children: React.Node,
```

Comment:
The new "this is stuff react knows how to render" type. See https://flow.org/en/docs/react/types/#toc-react-node for the documentation for this type.

Reply:
It's great that these aren't weird undocumented internals anymore.